lib/discordrb/data/user.rb

```diff
@@ -69,6 +63,7 @@ def initialize(data, bot)
       @discriminator = data['discriminator']
       @avatar_id = data['avatar']
       @roles = {}
+      @activities = Discordrb::ActivitySet.new([])
```

Comment:
`def initialize(activities = [])` in `ActivitySet` is fine, the empty array here is a little distracting
```suggestion
      @activities = Discordrb::ActivitySet.new
```